src/parquet/column/properties.h

```diff
@@ -82,28 +82,41 @@ static constexpr int64_t DEFAULT_PAGE_SIZE = 1024 * 1024;
 static constexpr bool DEFAULT_IS_DICTIONARY_ENABLED = true;
 static constexpr int64_t DEFAULT_DICTIONARY_PAGE_SIZE_LIMIT = DEFAULT_PAGE_SIZE;
 static constexpr int64_t DEFAULT_WRITE_BATCH_SIZE = 1024;
+static constexpr bool DEFAULT_COLLECT_STATISTICS = false;
```

Comment:
Are they enabled as default in `parquet-mr`? We should probably stick to the same defaults here. Collecting statistics costs a bit but will improve query times on the files a lot.